inference-engine/tests/functional/plugin/shared/include/behavior/set_preprocess.hpp

```diff
@@ -992,4 +992,38 @@ TEST_P(PreprocessDynamicallyInSetBlobTest, Infer) {
     }
 }
 
+TEST_P(PreprocessTest, InferWithRGB2BGRConversion) {
```

Comment:
I see you added parametrized test, but don't see cpu instance. Is it expected?

Reply:
Yes, it's expected. Instantiation is implemented here: 
`INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests, PreprocessTest,
                            ::testing::Combine(
                                    ::testing::ValuesIn(netPrecisions),
                                    ::testing::Values(CommonTestUtils::DEVICE_CPU),
                                    ::testing::ValuesIn(configs)),
                            PreprocessTest::getTestCaseName);`
in inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/set_preprocess.cpp:34